demisto_sdk/commands/content_graph/common.py

```diff
@@ -108,6 +108,60 @@ def server_name(self) -> str:
             return "classifier-mapper"
         return self.lower()
 
+    @property
+    def metadata_name(self) -> str:
+        if self == ContentType.SCRIPT:
+            return "automation"
+        elif self == ContentType.INDICATOR_TYPE:
+            return "reputation"
+        elif self == ContentType.LAYOUT:
+            return "layoutscontainer"
+        elif self == ContentType.TEST_PLAYBOOK:
+            return ContentType.PLAYBOOK.server_name
+        elif self == ContentType.MAPPER:
+            return "classifier"
+        return self.lower()
+
+    @property
+    def metadata_display_name(self) -> str:
+        if self == ContentType.SCRIPT:
+            return "Automation"
+        elif self == ContentType.INCIDENT_FIELD:
+            return "Incident Field"
+        elif self == ContentType.INCIDENT_TYPE:
+            return "Incident Type"
+        elif self == ContentType.INDICATOR_FIELD:
+            return "Indicator Field"
+        elif self == ContentType.INDICATOR_TYPE:
+            return "Reputation"
+        elif self == ContentType.LAYOUT:
+            return "Layouts Container"
+        elif self == ContentType.MAPPER:
+            return "Classifier"
+        elif self == ContentType.GENERIC_DEFINITION:
+            return "Generic Definition"
+        elif self == ContentType.GENERIC_FIELD:
+            return "Generic Field"
+        elif self == ContentType.GENERIC_MODULE:
+            return "Generic Module"
+        elif self == ContentType.GENERIC_TYPE:
+            return "Generic Type"
+        elif self == ContentType.PREPROCESS_RULE:
+            return "Pre Process Rule"
+        elif self == ContentType.PARSING_RULE:
+            return "Parsing Rule"
+        elif self == ContentType.MODELING_RULE:
+            return "Modeling Rule"
+        elif self == ContentType.CORRELATION_RULE:
+            return "Correlation Rule"
+        elif self == ContentType.XSIAM_DASHBOARD:
+            return "XSIAM Dashboard"
+        elif self == ContentType.XSIAM_REPORT:
+            return "XSIAM Report"
+        elif self == ContentType.XDRC_TEMPLATE:
+            return "XDRC Template"
+        return self
```

Comment:
```suggestion
    def metadata_display_name(self) -> str:
        return re.sub(r'([a-z](?=[A-Z])|[A-Z](?=[A-Z][a-z]))', r'\1 ', self.value)
```
note that it won't cover special cases like "Automation" and "Reputation" - is it necessary tho?

Reply:
Yes. but these cases can be excluded